content/documentation/pages/5-feature-guides/2-batch/6-composed-task.md

```diff
@@ -0,0 +1,643 @@
+- Composed Task Definition Name
+- Child Task App Name - The label or the name of the application to which the property should be applied.
+- Property Key - The key of the property that is to be set.
+
+To set a property `myproperty` for a child task named `task-a` in a composed task named `my-composed-task` for the following dsl:
+
+```
+task-a && task-b
+```
+
+It would look something like this:
+![Property Diagram](images/SCDF-composed-task-child-property-diagram.png)
+
+Similarly if I wanted to pass a deployer property the format would remain the same except the the property type would `deployer`, for example we need to set the `kubernetes.limits.cpu` for `task-a`:
+
+```
+    deployer.my-composed-task.task-a.kubernetes.limits.cpu=1000m
+```
+
+Launching a composed task and setting both `app` and `deployer` properties would be done in the following way using the UI:
+
+1. Launch composed task as shown below by pressing the `play` button next to the composed task definition that needs to be launched:
+   ![Specify Which Composed Task to Launch](images/SCDF-composed-task-child-property-example-launch.png)
+1. Set the properties as follows in the `Parameters` text box:
+   ![Launch the Composed Task](images/SCDF-composed-task-child-property-launch-props.png)
+1. Now press the `Launch the task` button.
+
+## Passing Properties to Composed Task Runner
+
+There are 3 components that make up the property:
+
+- Property Type - This tells Spring Cloud Data Flow if the property is either a `deployment` or a `app` type.
+  - Deployment properties - are instructions to the deployer responsible for deploying the child task.
+  - App properties - are properties passed directly the child task.
+- Composed Task Application Name - Unlike when passing properties to a child task where we used the composed task definition name we will use the name of the composed task runner app.
+- Property Key - The key of the property that is to be set.
+
+Let's launch a composed task where we want to pass the following properties to the `Composed Task Runner`:
+
+- increment-instance-enabled - an `app` property that allows a single `Composed Task Runner` instance to be re-executed without changing the parameters.
+- kubernetes.limits.cpu - a `deployer` property that sets the Kubernetes cpu limit for the composed task runner
+  Launching a composed task and setting both `app` and `deployer` properties for the `Composed Task Runner` would be done in the following way using the UI:
+
+1. Launch composed task as shown below by pressing the `play` button next to the composed task definition that needs to be launched:
+   ![Specify Which Composed Task to Launch](images/SCDF-composed-task-child-property-example-launch.png)
+1. Set the properties as follows in the `Parameters` text box:
+   ![Launch the Composed Task](images/SCDF-composed-task-child-property-launch-ctr-props.png)
+1. Now press the `Launch the task` button.
+
+# Launching Composed Task using RESTFul API
```

Comment:
For some reason this doesn't appear in the TOC.  Might be the first time we have too many entries in the right hand TOC and it doesn't support a scroll bar.